docs/_admin/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Project Administration
+order: -1
+---
+
+The web-platform-tests project is more than just tests! This section documents
+all the information necessary to administer the infrastructure which makes the
+project possible.
+
+## Secrets
+
+Some aspects of the infrastructure are only accessible to administrators. These
+include access credentials for third-party services (e.g. continuous
+integration providers), data used to prove identity (e.g. SSL certificates),
+and deployment information for the project's running systems (e.g.
+[w3c-test.org](https://w3c-test.org)).
+
+Project                   | Secret                   | Owners
+--------------------------|--------------------------|-------------
+[results-collection]      | root SSH keys            | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[results-collection]      | Password for app secrets | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[pull-requests-dashboard] | root SSH keys            | ???
+[pull-requests-dashboard] | Password for app secrets | boaz@bocoup.com, geoffers@gmail.com, jgraham@hoppipolla.co.uk, lukebjerring@google.com, mike@w3.org
+[web-platform-tests]      | TravisCI authorization   | ???
+
+Note that SSL certificates for all HTTPS-enabled domains are retrieved via
+[Let's Encrypt](https://letsencrypt.org/), so that data does not represent an
+explicitly-managed secret.
+
+## Third-party account owners
+
+- (unknown registrar): https://web-platform-tests.org
+  - ???
+- (unknown registrar): https://w3c-test.org
+  - ???
+- (unknown registrar): http://testthewebforward.org
```

Comment:
W3C Systeam.

Reply:
For this initial effort, I'd like to end up with a document that definitively identifies points-of-contact for each bit of secret information. Does the team have a shared e-mail address? Or could you tell me where I could find its member list?

Reply:
From https://www.w3.org/blog/systeam/, this appears to be web-human@w3.org . Does that sound right to you?

Reply:
```
+- (unknown registrar): https://w3c-test.org
+  - ???
+- (unknown registrar): http://testthewebforward.org
```
> From https://www.w3.org/blog/systeam/, this appears to be web-human@w3.org . Does that sound right to you?

Yes